### PR TITLE
Update start.rst

### PR DESCRIPTION
**Pull request by danbachar**
add virtualenv as a requirement

### CODE REVIEW

File: docs/apache-airflow/start.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ This quick start guide will help you bootstrap an Airflow standalone instance on
 
 .. note::
 
-   Successful installation requires a Python 3 environment. Starting with Airflow 2.7.0, Airflow supports Python 3.8, 3.9, 3.10 and 3.11.
+   Successful installation requires a Python 3 environment, along with virtualenv. Starting with Airflow 2.7.0, Airflow supports Python 3.8, 3.9, 3.10 and 3.11.
```
Comment (dirrao) [inline docs/apache-airflow/start.rst:27]: Not necessarily, virtualenv recommended way to install it.

**Comment by danbachar** [comment]
Without virtualenv the DAG mentioned in the tutorial does not exist.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Taragolis** [comment]
Virtual environment it is not a strict requirement it is only highly suggested into the non isolated environment. For example you could easily use Airflow which installed into the userspace into the Docker, that how it installed into the [Official Airflow Images](https://airflow.apache.org/docs/docker-stack/index.html).

There is difficult to recognise what is going wrong in your case it could be:
- Conflicting packages with system packages (if it installed as a root as system package)
- Conflicting packages into the userspaces
- Missing [`[core] load_examples`](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#load-examples) config
- etc

**Comment by potiuk** [comment]
I think the issue is different. You shoudl add note that some examples might not work if you do not install it with `[virtualenv]` extra. You do not **technically** need virtualenv to install airflow, but if you want to look at example dags and get them all working you need to install `airflow[virtualenv]`. There is a subtle difference between the two

https://airflow.apache.org/docs/apache-airflow/stable/extra-packages-ref.html#core-airflow-extras - airflow has different extras and virtualenv is one of them.

**Pull request closed without merging**